assets/js/components/product-category-control/style.scss

```diff
@@ -3,9 +3,54 @@
 		display: flex;
 		align-items: center;
 	}
+}
+
+.woocommerce-product-categories__item-label {
+	display: flex;
+	flex: 1;
+
+	.depth-1 & {
+		padding-left: $gap-small;
+	}
+
+	.depth-2 & {
+		padding-left: $gap-small * 2;
+	}
+
+	.depth-3 & {
+		padding-left: $gap-small * 3;
+	}
 
+	.depth-4 & {
```

Comment:
So thinking we just consider anything deeper too crazy and not worry about it? I'm fine with 4 levels.

Reply:
I've updated the CSS to max out at 5 levels deep, and everything after 5 levels will be the same level of indentation.